app/routes/_layout+/assets.$assetId.qr.tsx

```diff
@@ -66,7 +80,12 @@ export default function QRPreview() {
           </h6>
         </div>
         <figure className="qr-code flex  justify-center">
-          <img src={data.qr.src} alt={`${data.qr.size}-shelf-qr-code.png`} />
+          <ImagePreview
+            ref={imagePreviewRef}
+            qr={data.qr.src}
+            size={data.qr.size}
+            logo="/images/shelf-icon-28x.png"
```

Comment:
```suggestion
            logo="/images/shelf-icon-28px.png"
```

Reply:
Oh 😂, wow. Thank you man. You saved me.
A typo !